scripts/build.js

```diff
@@ -32,29 +34,31 @@ async function main() {
       // chokidar, which is quite convenient
       externals: ["chokidar"],
```

Comment:
chokidar is not a webpack dependencies since webpack 5. The watching is JS only and should bundle fine.

Reply:
I've removed this.